lore_extractor.py

```
@@ -0,0 +1,805 @@
+            return None
+
+        result = json.loads(output_text)
+
+        # Add turn number range to each new entry (use last turn number)
+        last_turn = narrations[-1].get("turn", 0) if narrations else 0
+        if "new_entries" in result:
+            for entry in result["new_entries"]:
+                entry["created_turn"] = last_turn
+                entry["last_updated"] = last_turn
+
+        return result
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)